kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -2903,18 +2903,43 @@ def _normalize_ha_network_groups(self, network_groups):
+            normalized_rules = []
+            for rule in rules:
+                if not isinstance(rule, dict):
+                    continue
+
+                resource = rule.get('resource')
+                if not isinstance(resource, string_types) or not resource:
+                    continue
+
+                try:
+                    weight = int(rule.get('weight', 0))
+                except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)